functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,130 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the length of a given argument in bits. It calculates the length of the argument and then returns the result in bits.
+
+Examples:
+
+```sql
+SELECT BIT_LENGTH("TiDB");
++----------+
+| BIT_LENGTH("TiDB") |
++----------+
+| 8 (T) + 8 (i) + 8 (D) + 8 (B) = 32 bits |
++----------+
+```
+
+```sql
+SELECT CustomerName, BIT_LENGTH(CustomerName) AS BitLengthOfName FROM Customers;
+| CustomerName|BitLengthOfName |
+|-------------|----------------|
+| Albert Einstein | 120 bits |
+| Robert Oppenheimer | 144 bits |
+```
+
+```sql
+SELECT BIT_LENGTH("PingCap 123");
++----------+
+| BIT_LENGTH("PingCap 123") |
++----------+
+| 8 (P) + 8 (i) + 8 (n) + 8 (g) + 8 (C) + 8 (a) + 8 (p) + 8 () + 8 (1) + 8 (2) + 8 (3) = 88 bits |
++----------+
+```
+
+> **Note:**
+>
+> The second example operates under the assumption that there is a database with a record titled `Customers` and a field inside titled `CustomerName`
```

Comment:
Not sure if word "record" is accurate, in my understanding, it is a table names "Customers" with "CustomerName" column.

Reply:
i'm sure it is  accurate. record (row) and field (column). admittedly, could have just used column and row tho

Reply:
changed  record to row, and field to column for simplicity, @yibin87